### PR TITLE
[15.0] [IMP] purchase_default_terms_conditions: changed field type Text to Html on note

### PR DESCRIPTION
**Pull request by Nikul-OSI**
@dreispt can you please take a look?

### CODE REVIEW

**Review by dreispt — APPROVED** [review]

**Review by sswapnesh — APPROVED** [review]

**Review by legalsylvain — CHANGES_REQUESTED** [review]
don't you have to set ``widget="html"`` in related views ?

**Comment by dreispt** [comment]
@legalsylvain Adding the widget is redundant
@Nikul-OSI Please fix pre-commit, thanks

**Comment by Nikul-OSI** [comment]
@dreispt Fixed pre-commit.

File: purchase_default_terms_conditions/views/res_partner_views.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <field name="priority">36</field>
         <field name="arch" type="xml">
             <group name="purchase" position="inside">
-                  <field name="purchase_note" />
+                  <field name="purchase_note" widget="html" />
```
Comment (dreispt) [inline purchase_default_terms_conditions/views/res_partner_views.xml]: @Nikul-OSI the widget is redundant as the field is already of type Html

**Pull request merged**